skrub/_gap_encoder.py

```diff
@@ -605,6 +605,10 @@ class GapEncoder(TransformerMixin, BaseEstimator):
        with the Kullback-Leibler divergence as loss, and a Gamma prior on `H`.
        We thus optimize `H` and `W` with the multiplicative update method.
 
+    "Gap" stands for "Gamma-Poisson", the families of distributions used in the
+    topic model to model the importance of topics in a document (Gamma), and
```

Comment:
"in the topic model to model" is a bit weird. Maybe "used in topic modelling to model"